app/src/main/java/moe/apex/rule34/image/ImageBoard.kt

```diff
@@ -319,7 +318,35 @@ object Yandere : ImageBoard {
             "q" -> ImageRating.QUESTIONABLE
             "e" -> ImageRating.EXPLICIT
             else -> ImageRating.UNKNOWN
-            // Sensitive does not exist for Yande.re
+            // Sensitive does not exist for Moebooru-based image boards
         }
     }
 }
+
```

Comment:
Breadboard generally uses two blank lines between top level definitions. Kotlin code guidelines prefer one. Up to you whether or not you want to change.